lib/install/management/appliance.go

```diff
@@ -1200,3 +1202,15 @@ func (d *Dispatcher) CheckServiceReady(ctx context.Context, conf *config.Virtual
 	}
 	return nil
 }
+
+// vchFolder returns the namespaced folder for the vch or an error.
+func vchFolder(op trace.Operation, sess *session.Session, conf *config.VirtualContainerHostConfigSpec) (*object.Folder, error) {
```

Comment:
Let's export this so we can use it in the ops-user permissions issue that I'm working on.